src/foam/parse/QueryParser.js

```diff
@@ -72,18 +72,18 @@ foam.CLASS({
       value: function(alt, literal, literalIC, not, notChars, optional, range,
           repeat, seq, seq1, str, sym) {
         return {
-          START: sym('query'),
-          query: sym('or'),
+          'START': sym('query'),
```

Comment:
Why add quotes?

Reply:
It was a rule in the linter that didn't like some of the keys in the object being quoted and some not being quoted. I think it's happy when they're consistent, not mixed. I can disable that rule if you want.

Reply:
https://eslint.org/docs/rules/quote-props

Reply:
Reverted